bower.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "SECP",
+  "name": "secp",
   "version": "0.0.0",
   "dependencies": {
-    "angular-bootstrap": "~0.10.0",
+    "angular-bootstrap": "^2.5.0",
```

Comment:
angular-bootstrap uses angular version 1.6.7. Can you please update every library with the prefix "angular" to version 1.6.7 i.e angular-route, angular, angular-mocks e.t.c. ?

This is to avoid conflicts when we run bower install. :)

Reply:
I will revert this back to 0.10.0, I did not use any of the new features from the latest version of bootstrap.